src/bucket.ts

```diff
@@ -989,6 +1004,140 @@ class Bucket extends ServiceObject {
     this.getFilesStream = paginator.streamify('getFiles');
   }
 
+  /**
+   * Set the object lifecycle rules for objects in this bucket.
+   *
+   * By default, the object lifecycle rule you provide to this method will be
+   * appended to any existing ones. To replace all existing rules, supply the
+   * `options` argument, setting `append` to `false`.
```

Comment:
Suggestion for first sentence. `By default, an Object Lifecycle Management rule provided to this method will be included to the existing policy.`